packages/echo-core/src/services/errorHandler/errorHandler.ts

```diff
@@ -0,0 +1,47 @@
+import { BaseError } from '@equinor/echo-base';
+import { AnalyticsModule } from '../../services/analytics/analyticsModule';
+
+type ErrorHandlerFunction = (exception: Error | BaseError | unknown, analyticsModule: AnalyticsModule) => void;
+interface ErrorHandler {
+    setErrorHandler: (handler: ErrorHandlerFunction) => void;
+    handleErrors: ErrorHandlerFunction;
+}
+
+// TODO: Remove when handleErrors is properly moved (and possibly reworked) to Core from EchoPedia
```

Comment:
I think we can remove this TODO comment. It's quite obvious when we rewrite errorHandler that  we will touch this code. TODO comments tends to build up over time, and suddenly we have several 100s. So we should use them very sparingly :)

Reply:
Yeah, I generally agree with the idea of less TODOs. I just wanted to emphasize that this is not meant as a final solution. I want to avoid where this part will be forgotten.

But maybe a TODO won't help on that.